rules/helm/package.tmpl.rb

```diff
@@ -0,0 +1,70 @@
+require "fileutils"
+require "open3"
+
+# Variables expanded by Bazel.
+package_dir = "[[package_dir]]"
+multipath_sep = "[[multipath_sep]]"
+tarsStr = "[[tars]]"
+generatedStr = "[[generated]]"
+helm = "[[helm]]"
+output_tgz = "[[output_tgz]]"
```

Comment:
Don't _really_ like the substitution (I'd prefer passing them in as environment variables to be evaluated by Ruby), but :man_shrugging:

Reply:
And I love the fact that I can avoid environment variables. :sweat_smile: